src/components/backend-ai-storage-proxy-list.ts

```diff
@@ -426,12 +427,16 @@ export default class BackendAIStorageProxyList extends BackendAIPage {
    * @param {object} rowData
    */
   controlRenderer(root, column?, rowData?) {
+    let unmatched: boolean;
+    if (this.agentDetail.cpu_util_live == null || !('current_mem' in this.agentDetail)|| !('live_stat' in this.agentDetail && 'node' in this.agentDetail.live_stat)){
```

Comment:
Also, please use a single `render()` function call, instead of using it in every control statement(`if`, `else`)

Reply:
I changed the code with adding a separate method `unmatchedFileSystem()` to determine if it is unmatched or not so that I can erase the control statement in `controlRenderer(root, column?, rowData?)`.